views/lexicon/tabs/progress_bars.html.twig

```diff
@@ -216,5 +216,32 @@
         </a>
 
     </div>
+    
+    <h2 class="heading">Stepped Progress Bar</h2>
+
+    <div class="stepped-progressbar">
+        <div class="step"><span class="step-box current"><span class="step-number">1</span></span></div>
```

Comment:
I still think this should be an `ol` rather than a collection of `divs` so that screen readers will announce it as a list with x number of items. Adding `role="progressbar"` with `aria-valuenow="1"`, `aria-valuemin="1"` & `aria-valuemax="4"` will also allow screenreaders and assistive technology to recognize it as a progress indicator and announce the current status, for example:

> Progress bar 1.

To make it sound a little nicer, you could add a visually hidden heading, `Step 1 of 4`, give it an `id` and add `aria-describedby="<heading-id>"`, which would give you:

> Progress bar, step 1 of 4, 1.

If we don't want screen readers to announce this at all (I'm not sure why we wouldn't), then we should add `aria-hidden="true"`